src/js/signin/containers/Main.jsx

```diff
@@ -45,11 +49,14 @@ class Main extends React.Component {
     this.loginUrlRequest = getLoginUrls(this.props.onUpdateLoginUrls);
   }
 
-  getVerifyUrl() {
-    if (!this.props.login.currentlyLoggedIn) return;
+  getVerifyUrl(forceRequest) {
+    if (!forceRequest && !this.props.login.currentlyLoggedIn) return;
```

Comment:
Why was this necessary?

Reply:
This is so that the method can be properly called in willReceiveProps if the loggedIn prop changes due to someone logging in, and if somebody lands on the verify page and does a page refresh